packages/airnode-deployer/src/utils/validation.test.ts

```diff
@@ -1,21 +1,34 @@
+import { join } from 'path';
+import { readFileSync } from 'fs';
 import { mockReadFileSync } from '../../test/mock-utils';
 import { loadConfig, loadTrustedConfig } from '@api3/airnode-node';
-import * as fixtures from '../../test/fixtures';
+import dotenv from 'dotenv';
+import { version as packageVersion } from '../../package.json';
 
 describe('config validation', () => {
-  it('loads the config without validation', () => {
-    const config = fixtures.buildConfig();
-    mockReadFileSync('config.json', JSON.stringify(config));
+  const exampleConfigPath = join(__dirname, '../../config/config.example.json');
```

Comment:
I agree that testing `config.example.json` is a good idea but I think that testing whatever comes from `fixtures.buildConfig()` is a valid configuration is also a good idea. That fixture is used all over the place and might easily become outdated. This would prevent that.

Reply:
> This would prevent that.

What would prevent it?

Reply:
Testing that the output of `fixtures.buildConfig()` is valid.

Reply:
I think those should import and use the `version` from `package.json`. I'll try to update them.